matchms/importing/load_adducts.py

```diff
@@ -27,3 +29,50 @@ def load_adducts(filename: str = None) -> Dict:
                          'adducts_negative': []}
 
     return known_adducts
+
+
+@lru_cache(maxsize=4)
+def load_known_adduct_conversions(filename: str = None) -> Dict:
+    """Load dictionary of known adduct conversions. Makes sure that file loading is cached.
+
+    Parameters
+    ----------
+    filename:
+        Json file containing adducts.
+    """
+    if filename is None:
+        adduct_conversions_file = os.path.join(os.path.dirname(__file__), "..", "data", "known_adduct_conversions.json")
+    else:
+        adduct_conversions_file = filename
+
+    if os.path.isfile(adduct_conversions_file):
+        known_adduct_conversions = json.load(open(adduct_conversions_file, "r"))
+    else:
+        print("Could not find json file with known adduct conversions.")
+        known_adduct_conversions = None
+
+    return known_adduct_conversions
+
+
+@lru_cache(maxsize=4)
+def load_adducts_table(filename: str = None) -> Dict:
```

Comment:
Same as `load_adducts_dict()`, please explain default file and format.

Reply:
Function is returning a pandas dataframe instead of dict.
```suggestion
def load_adducts_table(filename: str = None) -> pd.DataFrame:
```